tests/ui/ptr_as_ptr.fixed

```diff
@@ -0,0 +1,30 @@
+// run-rustfix
+
+#![warn(clippy::ptr_as_ptr)]
+
+fn main() {
```

Comment:
Also add tests for the MSRV of this lint, please. https://github.com/rust-lang/rust-clippy/pull/6507 is a great example how to add tests for the MSRV.